webpack.config.dev.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-undef */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const Dotenv = require('dotenv-webpack');
+
+process.env.NODE_ENV = 'development';
+
+module.exports = {
+	mode: 'development',
+	target: 'web',
+	entry: __dirname + '/src/index.js',
+	output: {
+		path: __dirname + '/public/',
+		publicPath: '/',
```

Comment:
Nice work,
However. it will be good if the `output dir` is *build* because we don't need to push compiled source,
consider this
```
const DIST_DIR = path.resolve(__dirname, 'build');
const SRC_DIR = path.resolve(__dirname, 'src');

const config = {
  entry: path.resolve(SRC_DIR, 'index.js'),
  output: {
    path: DIST_DIR,
    filename: 'bundle.js',
    publicPath: DIST_DIR,
  },
```

Reply:
Thank You for helping me!